tests/5.1/error/test_error_at_execution.c

```diff
@@ -0,0 +1,36 @@
+//===--- test_error_at_execution.c ----------------------------===//
+//
+//  OpenMP API Version 5.1 Aug 2021
+//
+//  Test uses error directive with at(execution) where if the error 
+//  condition is reached during runtime, it will print an error message.
+//  If error message does not print, then the test has failed. 
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+int errors, i;
```

Comment:
"errors" needs to be initialized to 0 and needs to be set when the error condition is met.